### PR TITLE
Fix: wpml default directory changing home url

### PR DESCRIPTION
**Pull request by Anindra123**

### CODE REVIEW

File: views/quiz/attempt-table.php [hunk 1]
```diff
@@ -162,8 +162,19 @@ function( $ans ) {
 									?>
 								<?php elseif ( 'details' == $key ) : ?>
 									<?php
-										$url   = add_query_arg( array( 'view_quiz_attempt_id' => $attempt->attempt_id ), tutor()->current_url );
-										$style = '';
+										$admin_url = add_query_arg(
```
Comment (harunollyo) [inline views/quiz/attempt-table.php:165]:
```php
$url_args   = array( 'view_quiz_attempt_id' => $attempt->attempt_id );
$front_url  = add_query_arg( $url_args, tutor()->current_url );
$admin_url  = add_query_arg( $url_args, admin_url( 'admin.php?page=tutor_quiz_attempts' ) );
```

File: views/qna/qna-table.php [hunk 2]
```diff
@@ -118,7 +118,21 @@
 									</div>
 								<?php elseif ( 'action' == $key ) : ?>
 									<div class="tutor-d-flex tutor-align-center tutor-justify-end tutor-gap-1">
-										<a href="<?php echo esc_url( add_query_arg( array( 'question_id' => $qna->comment_ID ), tutor()->current_url ) ); ?>" class="tutor-btn tutor-btn-outline-primary tutor-btn-sm">
+										<?php
```
Comment (harunollyo) [inline views/qna/qna-table.php:121]:
```php
$url_args   = array( 'view_quiz_attempt_id' => $attempt->attempt_id );
$front_url  = add_query_arg( $url_args, tutor()->current_url );
$admin_url  = add_query_arg( $url_args, admin_url( 'admin.php?page=question_answer' ) );
```

**Review by harunollyo — CHANGES_REQUESTED** [review]

File: classes/Assets.php [hunk 3]
```diff
@@ -156,6 +156,7 @@ public function get_default_localized_data() {
 		return array(
 			'ajaxurl'                      => admin_url( 'admin-ajax.php' ),
 			'home_url'                     => get_home_url(),
```
Comment (harunollyo) [inline classes/Assets.php:158]:
```php
rtrim( get_home_url(), '/')
```

**Review by harunollyo — CHANGES_REQUESTED** [review]

File: assets/react/v3/shared/atoms/WPEditor.tsx [hunk 4]
```diff
@@ -83,7 +83,7 @@ function editorConfig(
       toolbar2:
         'strikethrough hr | forecolor pastetext removeformat | charmap | outdent indent | undo redo | wp_help | fullscreen | tutor_button | undoRedoDropdown',
       content_css:
-        `${tutorConfig.home_url}/wp-includes/css/dashicons.min.css,${tutorConfig.home_url}/wp-includes/js/tinymce/skins/wordpress/wp-content.css,${tutorConfig.home_url}/wp-content/plugins/tutor/assets/lib/tinymce/light/content.min.css`,
+        `${tutorConfig.asset_base_url}/wp-includes/css/dashicons.min.css,${tutorConfig.asset_base_url}/wp-includes/js/tinymce/skins/wordpress/wp-content.css,${tutorConfig.asset_base_url}/wp-content/plugins/tutor/assets/lib/tinymce/light/content.min.css`,
```
Comment (sazedul-haque) [inline assets/react/v3/shared/atoms/WPEditor.tsx:86]: Use `site_url` here as we are reffering the wp-includes directory.

**Review by sazedul-haque — CHANGES_REQUESTED** [review]

**Review by sazedul-haque — APPROVED** [review]

**Pull request merged**